js/main.js

```diff
@@ -0,0 +1,23 @@
+// Функция, возвращающая случайное целое число из переданного диапазона включительно.
+// Учтите, что диапазон может быть только положительный, включая ноль. А также придумайте, как функция должна вести себя, если передать значение «до» меньшее, чем значение «от», или равное ему.
+
+function randomNumber(min, max) {
+  if (min <= 0 ) {
+    const ALERT_MESSAGE= 'Передайте номер больше 0 для min';
+    return ALERT_MESSAGE;
+  }
+  const INNER_NUMBER = Math.random()*(max-min) + min;
```

Comment:
Вокруг операндов надо ставить всегда пробелы, следи за этим
```suggestion
  const INNER_NUMBER = Math.random() * (max - min) + min;
```